lib/internal/bootstrap/web/exposed-window.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
```

Comment:
I think technically this file should be `exposed-window-or-worker` - workers don't have a window object, and things here are either defined as `Exposed=(Window,Worker)`, or through the `WindowOrWorkerGlobalScope` mixin.

Reply:
Yeah, makes sense. Thanks, fixed.